go/cli/mcap/cmd/info.go

```diff
@@ -35,6 +35,19 @@ func humanBytes(numBytes uint64) string {
 	return fmt.Sprintf("%.2f %s", displayedValue, prefixes[prefixIndex])
 }
 
+func getDurationNs(start uint64, end uint64) float64 {
+	// subtracts start from end, returning the result as a 64-bit float.
+	// float64 can represent the entire range of possible durations (-2**64, 2**64),
+	// albeit with some loss of precision.
+	diff := end - start
+	signMultiplier := 1.0
+	if start > end {
+		diff = start - end
+		signMultiplier = -1.0
+	}
+	return float64(diff) * signMultiplier
+}
```

Comment:
could we add a few quick unit tests for this, or for the printInfo function to prove that it handles the edge cases we want it to handle?

Reply:
yup, sure can.